src/api/introduction/pagination/index.mdx

```diff
@@ -17,22 +17,22 @@ To move between pages of a collection of records, use the links in the `next` an
 - ATTRIBUTE
   - DATA TYPE
   - DESCRIPTION
-- _links
+- \_links
```

Comment:
This shouldn't have a backtick

Reply:
Should this break anything? I think this was a prettier change, can revert it if we want to avoid introducing more characters.

Reply:
I don't think it will break anything. It might display differently though